components/chef-ui-library/package-lock.json

```diff
@@ -2063,7 +2063,7 @@
     },
     "es6-promisify": {
       "version": "5.0.0",
-      "resolved": "https://registry.npmjs.org/es6-promisify/-/es6-promisify-5.0.0.tgz",
+      "resolved": "http://registry.npmjs.org/es6-promisify/-/es6-promisify-5.0.0.tgz",
```

Comment:
This should be reverted--always want https in there. Recall there was an issue which we solved for automate-ui; looks like the same fix is needed in chef-ui-library. (IIRC it was a caching issue of sorts.)